dev/archery/archery/cli.py

```diff
@@ -284,9 +285,10 @@ def decorate_lint_command(cmd):
               help="Run IWYU on all C++ files if enabled")
 @click.option("-a", "--all", is_flag=True, default=False,
               help="Enable all checks.")
+@click.argument("path", required=False)
```

Comment:
Why do we need this?
Do we need to use custom path for `docs/`?

Reply:
No, I don't think so. We do want to point `archery lint --docs` at the docs subfolder but we can hard-code that in the code. No need yet to make it user-configurable.